### PR TITLE
Test Playbook for XSIAM 'Impossible Traveler Response'

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10770

## Description
Test playbook for the 'Impossible Traveler Response' XSIAM playbook.

### CODE REVIEW

File: Packs/Core/TestPlaybooks/Test_Playbook_-_Impossible_Traveler_Response.yml [hunk 1]
```diff
@@ -0,0 +1,2689 @@
...
+      "yes":
+      - "256"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: BlockIndicators
```
Comment (ssokolovich) [inline Packs/Core/TestPlaybooks/Test_Playbook_-_Impossible_Traveler_Response.yml:2206]: Does this key will be created with the provided IPs? I double-checked them with VT and seems like they r not malicious... maybe we should mark one of them as malicious to get this key?

File: Tests/conf.json [hunk 2]
```diff
@@ -21,6 +21,17 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "playbookID": "Test Playbook - Impossible Traveler Response",
+            "integrations": [
+                "VirusTotal (API v3)",
+                "Microsoft Graph"
```
Comment (ssokolovich) [inline Tests/conf.json:28]: do you need whois as well for geolocation?

Comment (ssokolovich) [reply Tests/conf.json:28]: Also, I see that you are referring to AD here - but you didn't mention it in this file 
image
image

Comment (TalNos) [reply Tests/conf.json:28]: removed the AD mentioned in the 'printerrorentry' tasks.

**Comment by TalNos** [comment]
All flows in the playbook end in 'Close alert', which closes the mock incident that runs the test playbook, so the playbook cannot be tested as once it closes, the test cannot continue.

**Pull request closed without merging**